### PR TITLE
gh 410 Change "entities" to "entity types"

### LINKED ISSUE #410: Change "entities" for "entity types" on custom filter
From feedback on the UI, the labels on the two drop-downs (Only include these entities/edges) are confusing and don't really convey the right message. Really we're narrowing down the element group (or type). The label on the drop-downs should reflect this.

### PR DESCRIPTION
**Pull request by g609bmsma**
Fixes #410 - Changed "entities" to "entity types" on view-builder.html only

### CODE REVIEW

File: ui/src/main/webapp/app/query/view-builder/view-builder.html [hunk 1]
```diff
@@ -8,42 +8,28 @@ <h3 class="md-title section-title">Add Filters</h3>
...
-                    <md-option ng-value="entity"
-                               ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
+                    <md-option ng-value="entity" ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
                         <div class="md-subhead">{{entity}}</div>
                         <div class="md-body-1">{{detail.description}}</div>
                     </md-option>
                 </md-select>
             </md-input-container>
             <md-input-container flex>
                 <label>Only include these edges</label>
```
Comment (d47853) [inline ui/src/main/webapp/app/query/view-builder/view-builder.html]: Could you apply the same change to the edges select component please?

Comment (g609bmsma) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: Could you clarify what you mean? Just putting it on the one line?

Comment (g609bmsma) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: Or changing "edges" to "edge types"?

Comment (d47853) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: Yeah changing 'edges' to 'edge types' in the &lt;label&gt; line and where it says `text="ctrl.createViewElementsLabel(ctrl.model.viewEdges, 'edges')"`. Just like what you did with the entities

Comment (g609bmsma) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: Done

**Pull request merged**